templates/image.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+
+export default class Image extends Component {
+	render () {
+
+		const createMarkup = () => {
```

Comment:
this function depends on variables set outside (and after) it - better to pass in the data as arguments